frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.tsx

```diff
@@ -202,16 +216,32 @@ const TeamEventCreditForm: React.FC = () => {
             Please include a picture of yourself (and others) and/or an email chain only if the
             former is not possible.
           </p>
-          <input
-            id="newImage"
-            type="file"
-            accept="image/png, image/jpeg"
-            defaultValue=""
-            value={image ? undefined : ''}
-            onChange={handleNewImage}
-          />
+          {images.map((image, i) => (
+            <div className="input_container" style={{ marginBottom: '10px' }} key={i}>
```

Comment:
nit: inline the styling here. also, what's className "input_container"? I don't see it defined in a CSS file